ansible/roles/sasl/tasks/main.yml

```diff
@@ -64,7 +84,7 @@
     - role::sasl
 
 - name: Enable and start SASL authentication daemon
-  service:
+  systemd_service:
```

Comment:
Couldn't this be acheived by running the service as a paricular user?

meaning

```py

become: true
become_user: sasl
```

However, this might imply that we need to add that user to sudoers, but only give it the righ to run that service only.


Also, can't we just modify the existing unit file ? you can use the `lineinfile` or `blockinfile` to replace the default one, and then just run the sevice.

Reply:
I also don't think that coupling to systemd is an issue, i don't see us using something else than systemd to run our services.

We will see what the others have to say about it.

Reply:
No, I believe that would change the user of systemd itself. @jchristgit left an excellent answer.

I don't have the contents of the existing unit file, and that approach seems more heavy-handed. Would we have to dupliucate the contents of that file and then append the user/group config to it? Adding `user.conf` seems neater.

Reply:
Ah nice i didn't know that!

Alright I see, thank boss